docs/source/v2.1/WaterCycle/index.rst

```diff
@@ -0,0 +1,16 @@
+E3SMv2.1 (Water Cycle)
+====================
+
+The simulation compaign for E3SMv2 (Water Cycle) was performed with two
```

Comment:
This needs to be modified as `The simulation campaign for E3SMv2.1  was performed with Low Resolution (LR) configuration`